packages/validation-manager/src/ERC7562Parser.ts

```diff
@@ -276,7 +276,9 @@ export class ERC7562Parser {
     recursionDepth: number,
     delegatecallStorageAddress: string
   ): void {
-    if (erc7562Call.to.toLowerCase() === this.entryPointAddress.toLowerCase()) {
+    if (erc7562Call.to.toLowerCase() === this.entryPointAddress.toLowerCase() ||
+      erc7562Call.to.toLowerCase() === this.senderCreatorAddress.toLowerCase()
```

Comment:
why is this needed? senderCreator is not the entrypoint 
(afaik, it doesn't cause an issue, because senderCreator's code is simple enoughL: doesn't have any storage access , and have requireFromEntryPoint on all methods)